src/hooks/useAsset.js

```diff
@@ -4,6 +4,51 @@ import { useSelector } from 'react-redux';
 import useAccountAssets from './useAccountAssets';
 import useUniswapAssetsInWallet from './useUniswapAssetsInWallet';
 import { AssetTypes } from '@rainbow-me/entities';
+import store from '@rainbow-me/redux/store';
+import { ETH_ADDRESS, ETH_ICON_URL } from '@rainbow-me/references';
+import { convertAmountToNativeDisplay } from '@rainbow-me/utilities';
+
+function getZeroEth() {
+  const {
+    data: { genericAssets },
+    settings: { nativeCurrency },
+  } = store.getState();
+
+  const { relative_change_24h, value } = genericAssets?.eth?.price || {};
+
+  return {
+    address: ETH_ADDRESS,
+    balance: {
+      amount: '0',
+      display: '0 ETH',
+    },
+    color: '#29292E',
+    decimals: 18,
+    icon_url: ETH_ICON_URL,
+    isCoin: true,
+    isPlaceholder: true,
+    isSmall: false,
+    name: 'Ethereum',
+    native: {
+      balance: {
+        amount: '0.00',
+        display: convertAmountToNativeDisplay('0.00', nativeCurrency),
+      },
+      change: relative_change_24h ? `${relative_change_24h.toFixed(2)}%` : '',
+      price: {
+        amount: value || '0.00',
+        display: convertAmountToNativeDisplay(
+          value ? value : '0.00',
+          nativeCurrency
+        ),
+      },
+    },
+    price: value,
+    symbol: 'ETH',
+    type: 'token',
+    uniqueId: ETH_ADDRESS,
```

Comment:
`uniqueId` has now been transformed to be of the form "address_network" (since we have L2 ETH assets that have the same address)